src/testutils/configurable_blockchain_tests.rs

```diff
@@ -1,8 +1,9 @@
+use assert_matches::assert_matches;
 use bitcoin::Network;
 
 use crate::{
-    blockchain::ConfigurableBlockchain, database::MemoryDatabase, testutils, wallet::AddressIndex,
-    Wallet,
+    blockchain::ConfigurableBlockchain, database::MemoryDatabase, error::Error,
+    signer::SignerError, testutils, wallet::AddressIndex, Wallet,
```

Comment:
I am seeing warning in the additions here as they aren't seemed to be used in this module..

Reply:
Removed. I didn't see warnings in IntelliJ, nor from cargo. 
What should I be doing/using to see these?

Reply:
If you are using a editor linting, you need to activate certain features to activate those segment of code.. This warning was visible if `test-blockchain` feature was activated.. Different part of the code base are activated with different features, so without the feature on that code will not be analyzed and warning wont be shown in the editor.. I don't know how to do it in IntelliJ, must be some settings that you have to add..

The other way is just build with various features like `cargo build --feature test-blockchain` etc..

Reply:
I use rust-analyzer with VSCode.. But other editors should work too..

Reply:
Found some useful config in IJ. Thanks.
Is this good to merge now?